README.md

```diff
@@ -1,2 +1,395 @@
+
+## Background
+
+### Why would I use Alpine?
+
+```mermaid
+flowchart LR
+    subgraph alpine["🖥️ Alpine"]
+        direction TB
+        subgraph resources[" &nbsp; 💪 Compute Resources &nbsp;"]
+            check_1["✅"]
+        end
+        subgraph time[" &nbsp; 🕑 Long-running Jobs &nbsp;"]
+            check_2["✅"]
+        end
+        subgraph collaborations[" &nbsp; 👥 Collaborations &nbsp;"]
+            check_3["✅"]
+        end
+    end
+
+style alpine fill:#ffffff,stroke:#444444;
+style check_1 fill:#D1FAE5,stroke:#D1FAE5;
+style check_2 fill:#D1FAE5,stroke:#D1FAE5;
+style check_3 fill:#D1FAE5,stroke:#D1FAE5;
+style alpine fill:#ffffff,stroke:#444444;
+```
+
+_Diagram showing common benefits of Alpine and HPC clusters._
+
+Alpine is a [High Performance Compute (HPC) cluster](https://en.wikipedia.org/wiki/High-performance_computing).
+HPC environments provide shared computer hardware resources like [memory](https://en.wikipedia.org/wiki/Computer_memory), [CPU](https://en.wikipedia.org/wiki/Central_processing_unit), [GPU](https://en.wikipedia.org/wiki/Graphics_processing_unit) or others to run performance-intensive work.
+Reasons for using Alpine might include:
+
+- __Compute resources:__ Leveraging otherwise cost-prohibitive amounts of memory, CPU, GPU, etc. for processing data.
+- __Long-running jobs:__ Completing long-running processes which may take hours or days to complete.
+- __Collaborations:__ Sharing a single implementation environment for reproducibility within a group (avoiding "works on my machine" inconsistency issues).
+
+### How does Alpine work?
+
+```mermaid
+flowchart LR
+    users["👥 Users"]
+    subgraph alpine["🖥️ Alpine"]
+        slurm["🗓️ Slurm"]
+        subgraph login_nodes["🔑 Login node(s)"]
+            acompile["📄 acompile"]
+            slurm_cmd["⌨️ Slurm cmd's"]
+        end
+        subgraph compute_nodes["⚙️ Compute node(s)"]
+            hardware["⚙️ Compute\nResources"]
+            modules["💿 Software via\nmodules pkg"]
+        end
+
+        subgraph storage["📂 Storage"]
+            local_storage["📁 Local Storage\n(sometimes temporary)"]
+        end
+    end
+    remote_storage["📁 External Storage\n(user specified / configured)"]
+
+    users --> | preconfigured\nSlurm access| acompile --> slurm
+    users --> | direct access | slurm_cmd --> slurm
+    slurm --> |"schedules\n(shared) use of"| hardware
+    slurm --> | provides\naccess to| modules --> | which may\n leverage| hardware
+    hardware --> | may deliver\nresults to| local_storage
+    hardware --> | or deliver\nresults to| remote_storage
+
+style alpine fill:#ffffff,stroke:#444444;
+style slurm fill:#F0F9FF,stroke:#075985;
+```
+
+_Diagram showing high-level user workflow and Alpine components._
+
+Alpine's compute resources are used through compute nodes in a system called [Slurm](https://github.com/SchedMD/slurm).
+Slurm helps coordinate shared and configurable access to the compute resources.
```

Comment:
Feel free to take this as-is, change it, or leave it out, but I think diving in a bit deeper into what Slurm is here would make concepts later in the document easier to explain. Here's my attempt; new terms are bolded for emphasis the around the first time they're defined:

```suggestion
Slurm is a system that a large number of users to run jobs on a cluster of computers; the system figures out how to use all the computers in the cluster to execute all the user's jobs fairly (i.e., giving each user approximately equal time and resources on the cluster). A *job* is a request to run something, e.g. a bash script or a program, along with specifications about how much RAM and CPU it needs, how long it can run, and how it should be executed.

Slurm's role in general is to take in a job (submitted via the `sbatch` command) and put it into a *queue* (also called a "partition" in Slurm). For each job in the queue, Slurm constantly tries to find a computer in the cluster with enough resources to run that job, then when an available computer is found runs the program the job specifies on that computer. As the program runs, Slurm records its output to files and finally reports the program's exit status (either completed or failed) back to the job manager.

Importantly, jobs can either be marked as *interactive* or *batch*. When you submit an interactive job, `sbatch` will pause while waiting for the job to start and then connect you to the program, so you can see its output and enter commands in real time. On the other hand, a *batch* job will return immediately; you can see the progress of your job using `squeue`, and you can typically see the output of the job in the folder from which you ran `sbatch` unless you specify otherwise.
```

Reply:
Cheers, this is fantastic! I'll commit this in.